e3sm_to_cmip/cmor_handlers/sitemptop.py

```diff
@@ -89,6 +91,6 @@ def handle(infiles, tables, user_input_path, **kwargs):
              'cell_bounds': ds.lon_bnds.values}]
     try:
         mpas.write_cmor(axes, ds, VAR_NAME, VAR_UNITS)
-    except Exception, as err:
```

Comment:
This comma was causing a syntax error, which was breaking the GH Actions test suite job. It is removed and the build should be working now.

The rest of the changes are from the code formatting tools.

Reply:
Thank you @tomvothecoder the comma was introduced right before pushing the code without testing again. My bad. Thank you for helping out.

Reply:
Thank you @tomvothecoder the comma was introduced right before pushing the code without testing again. My bad. Thank you for helping out.

Reply:
No problem, I add accidental characters like spaces too from time to time.